From 981641390daa1232ad6cc4b9269daaf2957b5735 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:59:29 -0400
Subject: [fs] btrfs: remove unused argument from select_reloc_root()

Message-id: <1407179109-13714-83-git-send-email-sandeen@redhat.com>
Patchwork-id: 87046
O-Subject: [RHEL7.1 PATCH 082/422] Btrfs: remove unused argument from select_reloc_root()
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Wang Shilong <wangsl.fnst@cn.fujitsu.com>

commit dc4103f933291cb1a2e6742c4db432e6ed337bae
Author: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
Date:   Thu Dec 26 13:10:49 2013 +0800

    Btrfs: remove unused argument from select_reloc_root()

    @nr is no longer used, remove it from select_reloc_root()

    Signed-off-by: Wang Shilong <wangsl.fnst@cn.fujitsu.com>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/relocation.c b/fs/btrfs/relocation.c
index 8cf99c4..277b8e3 100644
--- a/fs/btrfs/relocation.c
+++ b/fs/btrfs/relocation.c
@@ -2454,7 +2454,7 @@ static noinline_for_stack
 struct btrfs_root *select_reloc_root(struct btrfs_trans_handle *trans,
          struct reloc_control *rc,
          struct backref_node *node,
-         struct backref_edge *edges[], int *nr)
+         struct backref_edge *edges[])
 {
  struct backref_node *next;
  struct btrfs_root *root;
@@ -2496,7 +2496,6 @@ struct btrfs_root *select_reloc_root(struct btrfs_trans_handle *trans,
  if (!root)
   return NULL;
 
- *nr = index;
  next = node;
  /* setup backref node path for btrfs_reloc_cow_block */
  while (1) {
@@ -2643,7 +2642,6 @@ static int do_relocation(struct btrfs_trans_handle *trans,
  u32 blocksize;
  u64 bytenr;
  u64 generation;
- int nr;
  int slot;
  int ret;
  int err = 0;
@@ -2656,7 +2654,7 @@ static int do_relocation(struct btrfs_trans_handle *trans,
   cond_resched();
 
   upper = edge->node[UPPER];
-  root = select_reloc_root(trans, rc, upper, edges, &nr);
+  root = select_reloc_root(trans, rc, upper, edges);
   BUG_ON(!root);
 
   if (upper->eb && !upper->locked) {
-- 
1.7.1